spec/fixtures/sample.js

```diff
@@ -10,4 +10,20 @@ var quicksort = function () {
   };
 
   return sort(Array.apply(this, arguments));
-};
\ No newline at end of file
+};
+
+foo({
+    atIndentTwo: true
```

Comment:
> incorrect indentations like the following

I think I agree with you that there is an inconsistency, but to be fair: the problem with your first example is more than just indentation: if the callback argument appears on a new line, the object should start a new line as well, as shown in your second example.

But the example I gave is very common. Another even *more* common pattern is this (also from the node.js docs):

```js
fs.readFile('/etc/passwd', (err, data) => {
  // indented one level
});
```

We need to preserve the indentation levels in these cases.